client/payment-details/transaction-breakdown/index.tsx

```diff
@@ -0,0 +1,319 @@
+/** @format **/
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __ } from '@wordpress/i18n';
+import { filter, flatten, join } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useTimeline } from 'wcpay/data';
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	CardFooter,
+	Flex,
+	FlexItem,
+} from '@wordpress/components';
+import { TimelineItem, TimelineFeeRate } from 'wcpay/data/timeline/types';
+import Loadable, { LoadableBlock } from 'components/loadable';
+import { formatCurrency } from 'multi-currency/interface/functions';
+import './style.scss';
+
+interface PaymentTransactionBreakdownProps {
+	paymentIntentId: string;
+}
+
+const PaymentTransactionBreakdown: React.FC< PaymentTransactionBreakdownProps > = ( {
+	paymentIntentId,
+} ) => {
+	const { timeline, timelineError, isLoading } = useTimeline(
+		paymentIntentId
+	);
+
+	let captureEvents: TimelineItem[] = [];
+
+	if ( timeline ) {
+		captureEvents = filter( timeline, function ( item: TimelineItem ) {
+			return item.type === 'captured';
+		} );
+	}
+
+	let captureEvent: TimelineItem | undefined;
```

Comment:
Minor: This can be simplified by directly accessing captureEvents[0] and using optional chaining (?.) to avoid multiple explicit undefined checks. This keeps the code more concise while maintaining the same behavior.

```suggestion
const captureEvent = captureEvents[0] ?? undefined;

  if (!captureEvent?.transaction_details || !captureEvent?.fee_rates) {
	  return <div />;
  }


```